### ISSUE TITLE
Improve message for E0057 ('use of undeclared variable') for calls

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#684]
```js
undeclaredFunction();
```
Reports 'use of undeclared variable' (E0057), but it should say 'use of undeclared function'.

**Comment by singalhimanshu** [comment]
Are we planning to create another error for undeclared functions, that says 'use of undeclared function'?

**Comment by strager** [comment]
We could make another error, or we could parameterize the existing error. I don't know which approach we should take.
